app/views/pets/edit.html.erb

```diff
@@ -1,7 +1,3 @@
-<ul class="nav">
-  <li><%= link_to "All Pets", "/pets" %></li>
-  <li><%= link_to "All Shelters", "/shelters" %></li>
-</ul>
```

Comment:
navs are removed from all html files now, as we indicated needed to be done in last pull request